.github/workflows/generate_ipynb_files.yml

```diff
@@ -5,6 +5,7 @@ on:
     paths:
     - 'examples/*/*.py'
     - 'examples/*/build.sh'
```

Comment:
evidently this requires something like `- '.github/**' as well. The other yml as well